app/(gcforms)/[locale]/(form administration)/form-builder/[id]/settings/manage/notifications/NotificationsToggle.tsx

```diff
@@ -0,0 +1,72 @@
+"use client";
```

Comment:
You can remove "use client" declaration from this file because the parent component `ManageForm` is already declared as the boundary.  By including it in this file the linter complains about `toggleChecked` being a prop that is trying to cross the boundary that is not serializable.